--min_prev_trades  it not working

### System information
- **Have I written custom code (as opposed to using zenbot vanilla)**:NO
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Docker
- **Zenbot version** (commit ref, or version): #1526 
- **Zenbot branch**:unstable
- **NodeJS version**: v8.9.10
- **Python version (when using a python script)**:
- **Exact command to reproduce (include everything)**:
zenbot trade \
binance.NCASH-BTC \
--buy_max_amt=0.007 \
--strategy=macd \
--period=2m \
--min_periods=52 \
--ema_short_period=12 \
--ema_long_period=26 \
--signal_period=6 \
--up_trend_threshold=0 \
--down_trend_threshold=0 \
--overbought_rsi_periods=22 \
--overbought_rsi=76 \
--poll_trades=3000 \
--exact_buy_orders \
--exact_sell_orders \
--use_prev_trades \
--min_prev_trades=10 \
--sell_stop_pct=2 \
--buy_stop_pct=2 \
--max_sell_loss_pct=3 \
--max_buy_loss_pct=25 \
--profit_stop_enable_pct=9 \
--profit_stop_pct=0.5 \
--markdown_buy_pct=0.79 \
--order_poll_time=5000 \
--order_adjust_time=5000 \
--markup_sell_pct=0.84
- **Did I make any changes to conf-sample.js?**: NO

### Describe the problem
--min_prev_trades=10  it not working 

### Source code / Error logs
module.js:549
    throw err;
    ^

Error: Cannot find module '/app/extensions/exchanges/10/exchange'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at module.exports (/app/lib/engine.js:31:20)
    at Command.<anonymous> (/app/commands/trade.js:87:20)
    at Command.listener (/app/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
    at Command.parseArgs (/app/node_modules/commander/index.js:651:12)

--- a/commands/trade.js
+++ b/commands/trade.js
@@ -50,7 +50,7 @@ module.exports = function (program, conf) {
     .option('--exact_buy_orders', 'instead of only adjusting maker buy when the price goes up, adjust it if price has changed at all')
     .option('--exact_sell_orders', 'instead of only adjusting maker sell when the price goes down, adjust it if price has changed at all')
     .option('--use_prev_trades', 'load and use previous trades for stop-order triggers and loss protection')
-    .option('--min_prev_trades', 'minimum number of previous trades to load if use_prev_trades is enabled, set to 0 to disable and use trade time instead')
+    .option('--min_prev_trades <number>', 'minimum number of previous trades to load if use_prev_trades is enabled, set to 0 to disable and use trade time instead', Number, conf.min_prev_trades)
     .option('--disable_stats', 'disable printing order stats')
     .option('--reset_profit', 'start new profit calculation from 0')
     .option('--use_fee_asset', 'Using separated asset to pay for fees. Such as binance\'s BNB or Huobi\'s HT', Boolean, false)
@@ -78,7 +78,7 @@ module.exports = function (program, conf) {
       so.currency_increment = cmd.currency_increment
       so.keep_lookback_periods = cmd.keep_lookback_periods
       so.use_prev_trades = (cmd.use_prev_trades||conf.use_prev_trades)
-      so.min_prev_trades = parseInt(cmd.min_prev_trades||conf.min_prev_trades)
+      so.min_prev_trades = cmd.min_prev_trades
       so.debug = cmd.debug
       so.stats = !cmd.disable_stats
       so.mode = so.paper ? 'paper' : 'live'

